SU2_CFD/src/numerics/CNumerics.cpp

```diff
@@ -298,6 +356,76 @@ void CNumerics::GetInviscidProjFlux(const su2double *val_density,
   }
 
 }
+void CNumerics::GetInviscidProjFlux(const su2double *val_U,
+                                    const su2double *val_V,
+                                    const su2double *val_normal,
+                                    su2double *val_Proj_Flux) {
```

Comment:
And if you move the variables it would also make sense to move the methods, to try and keep things organized, we have a few `InviscidProjFlux` already.